src/rpc/rawtransaction_util.cpp

```diff
@@ -274,12 +274,13 @@ void SignTransaction(CMutableTransaction& mtx, const SigningProvider* keystore,
 
     // Script verification errors
     std::map<int, std::string> input_errors;
+    Optional<CAmount> inputs_amount_sum;
```

Comment:
Please [use `std::optional` in new code](https://github.com/bitcoin/bitcoin/blob/master/src/optional.h#L12).